README.md

```diff
@@ -32,8 +32,8 @@ On hovering over a cell:
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://dmitruz.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://dmitruz.github.io/layout_calendar/report/html_report/)
```

Comment:
Provide these links to PR description 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/10644762-30c4-4b0c-ac04-f934b34f7efb)